backend/fms_core/services/sample_next_step.py

```diff
@@ -48,4 +48,125 @@ def queue_sample_to_study_workflow(sample_obj: Sample, study_obj: Study, order:
                                                              study=study_obj)
         except Exception as err:
             errors.append(err)
-    return sample_next_step, errors, warnings
\ No newline at end of file
+    return sample_next_step, errors, warnings
+
+def dequeue_sample_from_specifc_step_study_workflow(sample_obj: Sample, study_obj: Study, order: int) -> Tuple[Union[int, None], List[str], List[str]]:
+    """
+    Deletes a SampleNextStep instance to indicate the removal of a sample from a workflow at a specific step. 
+    Args:
+        `sample_obj`: Sample instance to be dequeued from the study workflow.
+        `study_obj`: Study instance currently associated to the sample.
+        `order`: Positive integer indicating the current position of the sample in the workflow.
+                 
+    Returns:
+        Tuple containing the number of SampleNextStep instances deleted, the error messages and the warning messages. 
```

Comment:
Not much sense in having a number when you can either have one or zero entries removed.